src/certificates/data/thunks.js

```diff
@@ -21,3 +32,31 @@ export function fetchCertificates(courseId) {
     }
   };
 }
+
+export function createCourseCertificate(courseId, certificates) {
```

Comment:
[important]: Now if I upload a signature for a certificate and click the "Upload" button with the Internet turned off, we do not receive an error notification

<img width="1560" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/8d3e6ff1-6548-4c0a-a225-64009e330950">